backend/internal/middleware/cors.go

```diff
@@ -7,7 +7,7 @@ import (
 
 func Cors() fiber.Handler {
 	return cors.New(cors.Config{
-		AllowOrigins:     "http://localhost:3000",
+		AllowOrigins:     "http://localhost:3000,https://gitmarks.org",
```

Comment:
should only allow localhost in dev env, big security vulnerability in production